orchestra/cmds/shell.py

```diff
@@ -46,13 +46,7 @@ def handle_shell(args):
 
     if command:
         script_to_run = " ".join(shlex.quote(c) for c in command)
-        p = _run_script(
-            script_to_run,
-            environment=env,
-            strict_flags=False,
-            cwd=cd_to,
-            loglevel="DEBUG",
-        )
+        p = _run_script(script_to_run, environment=env, strict_flags=False, cwd=cd_to, loglevel="DEBUG", exec_=True)
```

Comment:
We should not be using internal methods.
Can you make a new method in `orchestra/actions/util/__init__.py`?

Also, boolean arguments are mostly harmful and confusing. Make a wrapper with a dedicated name: `exec_script`.

Reply:
Fixed